leshan-server-redis/src/main/java/org/eclipse/leshan/server/redis/serialization/EndpointContextSerDes.java

```diff
@@ -78,10 +75,12 @@ public static JsonObject serialize(EndpointContext context) {
         /** copy the attributes **/
         Map<String, Object> attributes = context.entries();
         if (!attributes.isEmpty()) {
-            DatagramWriter writer = new DatagramWriter();
-            SerializationUtil.write(writer, attributes);
-            String base64 = StringUtil.byteArrayToBase64(writer.toByteArray());
-            peer.set(KEY_ATTRIBUTES, base64);
+            JsonObject attContext = Json.object();
+            for (String key : attributes.keySet()) {
+                // write all values as string
+                attContext.set(key, attributes.get(key).toString());
```

Comment:
Looking at deserialization, I saw that byte[] are decoded as hexString.
I'm not sure but it seems here this is not the case ?

Reply:
Yep. I adapted it to:

```
                Object value = attributes.get(key);
                if (value instanceof Bytes) {
                    value = ((Bytes) value).getAsString();
                }
                attContext.set(key, value.toString());
``` 

I will push after running the tests.